connectors/sources/confluence.py

```diff
@@ -221,17 +232,27 @@ async def api_call(self, url):
             response: Client response
         """
         self._logger.debug(f"Making a GET call for url: {url}")
-        try:
-            async with self._get_session().get(
-                url=url,
-                ssl=self.ssl_ctx,
-            ) as response:
-                yield response
-        except ServerDisconnectedError:
-            await self.close_session()
-            raise
-        except ClientResponseError as exception:
-            await self._handle_client_errors(url=url, exception=exception)
+        if url != os.path.join(self.host_url, PING_URL):
+            self.api_total_count += 1
+        while True:
```

Comment:
Is this necessary? In the success case, we hit the `break` statement, and in all the `except` clauses we will end up re-raising (had to double check for `self._handle_client_errors()`, but I don't see a scenario where a `raise` does not happen). So in what case do we ever loop?